src/NemoStuff.jl

```diff
@@ -536,10 +536,6 @@ function change_base_ring(p::MPolyRingElem{T}, g, new_polynomial_ring) where {T<
     return finish(M)::elem_type(new_polynomial_ring)
 end
 
-function leading_coefficient(f::Generic.MPoly)
-    return f.coeffs[1]
-end
-
```

Comment:
This fails for the zero polynomial. AA already contains a better implementation.